enterprise/server/sociartifactstore/sociartifactstore.go

```diff
@@ -105,7 +102,6 @@ func NewSociArtifactStore(env environment.Env) (error, *SociArtifactStore) {
 		cache:     env.GetCache(),
 		blobstore: env.GetBlobstore(),
 		env:       env,
```

Comment:
you'll want to skip setting up the artifact store if deduper is not available in the environment (if redis is not configured)

Reply:
fixed